modules/swagger-codegen/src/main/resources/ruby/README.mustache

```diff
@@ -84,6 +84,18 @@ end
 {{/-first}}{{/operation}}{{/operations}}{{/-first}}{{/apis}}{{/apiInfo}}
 ```
 
+### Region
+
+Smooch is available in the following [regions](https://docs.smooch.io/rest/#region). The US region will be used by default. To target any other region, specify the region [API host](https://docs.smooch.io/rest/#region) in `config.host`. For example:
+
+```ruby
+{{{moduleName}}}.configure do |config|
+  config.api_key['Authorization'] = '<jwt>'
+  config.api_key_prefix['Authorization'] = 'Bearer'
+  config.host = 'api.eu-1.smooch.io'
```

Comment:
no `https://` here?

Reply:
Nope. Ruby's concept of `host` is in the URL portion sense, whereas python's is in the URL root sense. /shrug